src/JSInterop/Microsoft.JSInterop/src/IJSStreamReference.cs

```diff
@@ -16,15 +16,16 @@ public interface IJSStreamReference : IAsyncDisposable
         /// <summary>
         /// Length of the <see cref="Stream"/> provided by JavaScript.
         /// </summary>
-        public long Length { get; }
+        long Length { get; }
 
         /// <summary>
         /// Opens a <see cref="Stream"/> with the <see cref="JSRuntime"/> for the current data reference.
         /// </summary>
         /// <param name="maxAllowedSize">Maximum number of bytes permitted to be read from JavaScript.</param>
-        /// <param name="maxBufferSize">Maximum number of bytes that are allowed to be buffered.</param>
+        /// <param name="pauseIncomingBytesThreshold">The number of unconsumed bytes to accept from JS before blocking. A value of zero prevents JS from blocking, allowing .NET to receive an unlimited number of bytes.</param>
```

Comment:
We can't have this, otherwise you can flood the server memory with data

Reply:
https://github.com/dotnet/aspnetcore/pull/33916#discussion_r660800108